src/components/PostsList/PostsList.tsx

```diff
@@ -1,37 +1,83 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './PostsList.scss';
```

Comment:
Consider to use [`styled-components`](https://styled-components.com/) or [`css modules`](https://github.com/css-modules/css-modules)
1. Directly importing css file is not a good practice fro component-level styling 
2. Styles are not scoped to the component (global scoped)
3. Harder to achieve component-level styling.